src/easylogging++.h

```diff
@@ -233,7 +231,9 @@ ELPP_INTERNAL_DEBUGGING_OUT_INFO << ELPP_INTERNAL_DEBUGGING_MSG(internalInfoStre
 #  define STRCPY(a, b, len) strcpy(a, b)
 #endif
 // Compiler specific support evaluations
-#if ((!ELPP_MINGW && !ELPP_COMPILER_CLANG) || defined(ELPP_FORCE_USE_STD_THREAD))
```

Comment:
120th column but you can ignore it as our release script uses `astyle` anyway to do this. If you look at current released version it is consistent but develop may be inconsistent